isolate/package.json

```diff
@@ -46,7 +46,7 @@
     "symbol-observable": "^1.2.0",
     "ts-node": "^7.0.0",
     "tslint": "^5.11.0",
-    "typescript": "3.0.x"
+    "typescript": "^3.1.3"
```

Comment:
Can we first update to TS 3.0 then later to 3.1? I think 3.1 introduced a few breaking changes.

Reply:
I read through it and i didnt spot any

Reply:
Had a look, the only breaking changes are
- Some vendor-specific types are removed from lib.d.ts (ie element.msMatchesSelector, not relevant for the user)
- Some [typeof](https://github.com/Microsoft/TypeScript/wiki/Breaking-Changes#narrowing-functions-now-intersects--object-and-unconstrained-generic-type-parameters) changes that also only affect us internally because they are backwards compatible with 3.0